corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -97,6 +97,13 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
           {% trans "Add Case Property Group" %}
         </button>
       </form>
+      <p class="text-muted" style="margin-top: .5rem;">
```

Comment:
`class="help-block"` does greyed out text and top and bottom margins

Reply:
Ah - perfect! Thanks. 

https://github.com/dimagi/commcare-hq/pull/29042/commits/b77fd92678699227e5979a8875dbd1e4097ab274